app/controllers/organizations_controller.rb

```diff
@@ -0,0 +1,139 @@
+    else
+      redirect_to organizations_invitations_path, notice: "Something went wrong !"
+    end
+  end
+
+  def my_organizations
+    authorize Organization, :my_organizations?
+    @my_organizations = UserOrganization.where(user: current_user, invitation: 'accepted')
+
+  end
+  def make_admin
+    @organization = Organization.find(params[:id])
+    authorize @organization, :make_admin?
+    @new_admin = User.find(params[:user_id])
+    update_record = UserOrganization.find_by(user: @new_admin, organization: @organization)
+    if update_record.update(role: 'admin')
+      redirect_to organization_path(@organization.id), notice: "Maiden admin successfully"
+    else
+      redirect_to organization_path(@organization.id), notice: "Something went wrong"
+    end
+  end
+
+  def tradings
+    @organization = Organization.find(params[:id])
+    authorize @organization, :tradings?
+    current_month = Time.current.beginning_of_month..Time.current.end_of_month
+    @tradings = Trading.where(organization: @organization, created_at: current_month).includes(:user).includes(:organization)
+    @comments = Comment.where(organization: @organization, created_at: current_month).includes(:organization)
+  end
+  def add_trading
+    @organization = Organization.find(params[:id])
+    authorize @organization, :add_trading?
+
+  end
+
+  def create_trading
+    @organization = Organization.find(params[:id])
+    authorize @organization, :create_trading?
+    if Trading.create(user: current_user, organization: @organization, amount: params[:amount] )
+      redirect_to add_trading_organization_path, notice: "Transaction record added successfully !"
+    else
+      redirect_to add_trading_organization_path, notice: "Failed to add transaction, something went wrong !"
+    end
+  end
+
+  def delete_trading
+    @organization = Organization.find(params[:id])
+    authorize @organization, :delete_trading?
+    if Trading.destroy(params[:trading_id])
+      redirect_to tradings_organization_path, notice: "Deleted trading record successfully !"
+    else
+      redirect_to tradings_organization_path, notice: "Something went wrong !"
+    end
+  end
+
+  def add_comment
+    @organization = Organization.find(params[:id])
+    authorize @organization, :add_comment?
+    if Comment.create(user: current_user, organization: @organization, body: params[:comment])
+      redirect_to tradings_organization_path, notice: "Comment added successfully !"
+    else
+    redirect_to tradings_organization_path, notice: "Something went wrong !"
+    end
+  end
+
+  def delete_comment
+    @organization = Organization.find(params[:id])
+    authorize @organization, :delete_comment?
+    if Comment.destroy(params[:comment_id])
+      redirect_to tradings_organization_path, notice: "Comment deleted successfully !"
+    else
+      redirect_to tradings_organization_path, notice: "Something went wrong !"
+    end
+  end
+
```

Comment:
Unnecessary extra line

Reply:
Query: bhaiya shouldn't there be an extra line as it is the ending of a function?

Reply:
@mai-wtag we dont need to add an extra line after the last function. extra lines should be added between two functions. For Code styles I'd suggest to have a look at here - https://docs.rubocop.org/rubocop/usage/basic_usage.html

Reply:
okay bhaiya, removing all extra line after the last function

Reply:
done bhaiya